imports/ui/pages/Community.js

```diff
@@ -20,31 +17,35 @@ const MessengerWithData = withTracker(props => {
   };
 })(Messenger);
 
-const RoomBlock = ({ activeRoom, list, onChange, onAdd, openRightBlock }) => {
+const RoomBlock = ({ activeRoom, list, onChange, onAdd, openRightBlock, translate }) => {
```

Comment:
Let's move `RoomBlock`  to separate file. Do not pass `translate` as prop - use HOC instead.